.markdownlint.yaml

```diff
@@ -0,0 +1,45 @@
+comment: 'Rules for PingCAP docs-cn'
+
+# disable all by default
+default: false
+
+heading-increment: true
+heading-style:
+  style: atx
+no-trailing-spaces: true
+no-hard-tabs:
+  code_blocks: false
```

Comment:
We also need this check in code blocks.

Reply:
We need to make sure that people use spaces instead of tabs in code blocks too. Please change this setting, thanks! @aylei